net/aria2/files/aria2.init

```diff
@@ -323,8 +323,7 @@ aria2_start() {
 	fi
 
 	procd_open_instance "${NAME}.${section}"
-	procd_set_param command "$PROG"
-	procd_append_param command --conf-path="${config_file}"
+	procd_set_param command $PROG --conf-path=${config_file}
```

Comment:
why this change?

Reply:
I think using procd_set_param is enough.